server/services/doc.js

```diff
@@ -0,0 +1,177 @@
+//  Defines various services for the role object
+
+const Sequelize = require('sequelize');
+// Require sequelize from the connection settings
+const sequelize = require('../../settings/connect');
+const jwt = require('jsonwebtoken');
+
+const secret = process.env.SECRET || 'documentmanagement';
+
+// Call the doc model and specify the arguments.
+const Doc = require('../../app/models/doc')(sequelize, Sequelize);
+const User = require('../../app/models/user')(sequelize, Sequelize);
+
+sequelize.sync({});
+
+/**
+ * Create a new document
+ * @param {object} req
+ * @param {function} res // Callback
+ * @returns {boolean} true if created, false otherwise.
+ */
+module.exports.createDocument = (req, res) => {
+  const jwtcode = req.headers.authorization;
+  const token = jwt.verify(jwtcode, secret);
```

Comment:
is this not a repetition of what `validate.validateToken()`. And if you're using it as a middleware then you don't need to verify, it will only proceed to this if `next()` is called
[http://expressjs.com/en/guide/using-middleware.html](http://expressjs.com/en/guide/using-middleware.html)